eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -2,7 +2,7 @@ parameters:
 - name: PackageName
   type: string
   default: 'not-specified'
-- name: ServiceName
```

Comment:
Perhaps as a transition you have both ServiceName and ServiceDirectory and you do something like `coalesce(parameters.ServiceDirectory, parameters.ServiceName)`, and then once you have converted all the usages you can remove the ServiceName parameter.